packages/lesswrong/components/messaging/NewConversationButton.tsx

```diff
@@ -18,20 +19,41 @@ const NewConversationButton = ({ user, currentUser, children, templateCommentId
     fragmentName: 'newConversationFragment',
   });
   const { history } = useNavigation();
-  const newConversation = useCallback(async () => {
+  const newConversation = useCallback(async () =>  {
     const alignmentFields = forumTypeSetting.get() === 'AlignmentForum' ? {af: true} : {}
 
     const response = await createConversation({
-      data: {participantIds:[user._id, currentUser._id], ...alignmentFields},
+      data: {participantIds:[user._id, currentUser?._id], ...alignmentFields},
     })
     const conversationId = response.data.createConversation.data._id
     const search = templateCommentId ? {search:`?${qs.stringify({templateCommentId: templateCommentId})}`} : {}
     history.push({pathname: `/inbox/${conversationId}`, ...search})
   }, [createConversation, user, currentUser, history, templateCommentId]);
 
+  // Checks if unnamed conversation between the two users exists
+  const terms: ConversationsViewTerms = {view: 'userConversations', userId: currentUser?._id, showArchive: true};
+  const { results } = useMulti({  
+    terms,
+    collectionName: "Conversations",
+    fragmentName: 'conversationsListFragment',
+    fetchPolicy: 'cache-and-network',
+    limit: 200,
```

Comment:
I haven't yet reviewed the whole PR or thought about how it fits together (later today I will try think more holistically about it). I want to note that it's been awhile since we had much open-source engagement, or updated our issue tracker, and there's some chance this is issue was out of date in some way. 

But, assuming this basic design choice makes sense, my guess is that this should have a limit of 1 rather than 200 (ie. you only need to check if a single conversation exists). You can create a view that filters for conversations that don't a title.

Reply:
I have tried to create a new view like this:
`Conversations.addView("userUntitledConversations", function (terms: ConversationsViewTerms) {
  const showArchivedFilter = terms.showArchive ? {} : {archivedByIds: {$ne: terms.userId}}
  return {
    selector: {participantIds: terms.userId, messageCount: {$gt: 0}, title: null, ...showArchivedFilter},
    options: {sort: {latestActivity: -1}}
  };
});`
but the title selector doesn't work. How do I need to change it so that it only collects all untitled conversations?

Reply:
Ah, "title: null" doesn't do the thing you want (it's a bit confusing, I think it removes it rom the selector rather than selecting for things with title = null. There's some intermediate stuff in our view handling where null is a special case)

Instead you want to import "viewFieldNullOrMissing"

```
import { viewFieldNullOrMissing } from '../../vulcan-lib';
```
and then make the selector:

```
selector: {participantIds: terms.userId, messageCount: {$gt: 0}, title: viewFieldNullOrMissing, ...showArchivedFilter},
```

However, we'll take it from here and make the further changes. Thanks for the help!